staker-rewards/rewards.js

```diff
@@ -77,12 +78,24 @@ function validateIntervalTimestamps(interval) {
   console.log(clc.green(`Interval end: ${endDate.toISOString()}`))
 }
 
-function calculateOperatorsRewards(cache, interval) {
+async function calculateOperatorsRewards(context, interval) {
+  const {cache} = context
+
   const slaCalculator = SLACalculator.initialize(cache, interval)
 
   const operatorsRewards = []
 
   for (const operator of getOperators(cache)) {
+    const isFraudulent = await isOperatorKeepECDSAFraudulent(context, operator)
```

Comment:
I'm wondering if we shouldn't just pass the `isFraudulent` flag as a new column in the `OperatorRewards` table. This way we will have a clean summary of each operator and we can also easily detect the cause if an operator doesn't qualify for the reward.

Reply:
Sure, good idea. But do we want to execute `calculateOperatorSLA` for fraudulent operator in case of a fraud?

Reply:
I think it doesn't hurt to calculate SLA for such operators to display them in the result table. It is not a very time-consuming process for a single operator and we'll have all the information in place in case we need it.